opensrp-anc/src/main/java/org/smartregister/anc/library/interactor/RegisterInteractor.java

```diff
@@ -62,103 +62,82 @@ public void onDestroy(boolean isChangingConfiguration) {
     public void getNextUniqueId(final Triple<String, String, String> triple,
                                 final RegisterContract.InteractorCallBack callBack) {
 
-        Runnable runnable = new Runnable() {
-            @Override
-            public void run() {
-                UniqueId uniqueId = getUniqueIdRepository().getNextUniqueId();
-                final String entityId = uniqueId != null ? uniqueId.getOpenmrsId() : "";
-                appExecutors.mainThread().execute(new Runnable() {
-                    @Override
-                    public void run() {
-                        if (StringUtils.isBlank(entityId)) {
-                            callBack.onNoUniqueId();
-                            PullUniqueIdsServiceJob.scheduleJobImmediately(
-                                    PullUniqueIdsServiceJob.TAG); //Non were found...lets trigger this againz
-                        } else {
-                            callBack.onUniqueIdFetched(triple, entityId);
-                        }
-                    }
-                });
-            }
+        Runnable runnable = () -> {
+            UniqueId uniqueId = getUniqueIdRepository().getNextUniqueId();
+            final String entityId = uniqueId != null ? uniqueId.getOpenmrsId() : "";
+            appExecutors.mainThread().execute(() -> {
+                if (StringUtils.isBlank(entityId)) {
+                    callBack.onNoUniqueId();
+                    PullUniqueIdsServiceJob.scheduleJobImmediately(
+                            PullUniqueIdsServiceJob.TAG); //Non were found...lets trigger this againz
+                } else {
+                    callBack.onUniqueIdFetched(triple, entityId);
+                }
+            });
         };
 
         appExecutors.diskIO().execute(runnable);
     }
 
     @Override
-    public void saveRegistration(final Pair<Client, Event> pair, final String jsonString, final boolean isEditMode,
-                                 final RegisterContract.InteractorCallBack callBack) {
-
-        Runnable runnable = new Runnable() {
-            @Override
-            public void run() {
-                saveRegistration(pair, jsonString, isEditMode);
-                appExecutors.mainThread().execute(new Runnable() {
-                    @Override
-                    public void run() {
-                        callBack.onRegistrationSaved(isEditMode);
-                    }
-                });
-            }
+    public void saveRegistration(android.support.v4.util.Pair<Client, Event> pair, String jsonString, boolean isEditMode, RegisterContract.InteractorCallBack callBack) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'android.support.v4.util.Pair' due to existing import 'android.support.v4.util.Pair'](https://app.codacy.com/app/OpenSRP/opensrp-client-anc/pullRequest?prid=4499997)